cl/lib/elasticsearch_utils.py

```diff
@@ -55,6 +55,48 @@ def build_daterange_query(
     return []
 
 
+def make_es_boost_list(fields: Dict[str, float]) -> list[str]:
+    """Constructs a list of Elasticsearch fields with their corresponding
+    boost values.
+
+    :param fields: A dictionary where keys are field names and values are
+    the corresponding boost values.
+    :return: A list of Elasticsearch fields with boost values formatted as 'field_name^boost_value'.
+    """
+    boosted_fields = []
+    for k, v in fields.items():
+        boosted_fields.append(f"{k}^{v}")
+    return boosted_fields
+
+
+def add_fields_boosting(cd: CleanData) -> list[str]:
+    """Applies boosting to specific fields according the search type.
+
+    :param cd: The user input CleanedData
+    :return: A list of Elasticsearch fields with their respective boost values.
+    """
+    # Apply standard qf parameters
+    qf = BOOSTS["qf"][cd["type"]].copy()
+    if cd["type"] in [SEARCH_TYPES.ORAL_ARGUMENT]:
+        # Give a boost on the case_name field if it's obviously a case_name
+        # query.
+        vs_query = any(
+            [
+                " v " in cd.get("q", ""),
```

Comment:
We can avoid repeating the `get` operation using a `variable` like the following:

```python
query = cd.get("q", "")
vs_query = any(
    [
        " v " in query,
        " v. " in query,
        " vs. " in query,
        " vs " in query,
    ]
)
```

Reply:
Sure, we should apply this change too.